backend/coins/btc/handlers/handlers.go

```diff
@@ -196,31 +196,45 @@ func (handlers *Handlers) getAccountTransactions(_ *http.Request) (interface{},
 }
 
 func (handlers *Handlers) postExportTransactions(_ *http.Request) (interface{}, error) {
+	type result struct {
+		Success      bool   `json:"success"`
+		Path         string `json:"path"`
+		ErrorMessage string `json:"errorMessage"`
+	}
 	name := fmt.Sprintf("%s-%s-export.csv", time.Now().Format("2006-01-02-at-15-04-05"), handlers.account.Config().Code)
 	downloadsDir, err := config.DownloadsDir()
 	if err != nil {
-		return nil, err
+		handlers.log.WithError(err).Error("error exporting account")
+		return result{Success: false, ErrorMessage: err.Error()}, nil
+	}
+	suggestedPath := filepath.Join(downloadsDir, name)
+	path := handlers.account.Config().GetSaveFilename(suggestedPath)
+	if path == "" {
+		return nil, nil
```

Comment:
in account.tsx there's this logic:

```js
        accountApi.exportAccount(this.props.code)
            .then(({ success, path, errorMessage }) => {
                if (success) {
                    this.setState({ exported: path });
                } else {
                    alertUser(errorMessage);
                }
            })
            .catch(console.error);
```

but if export is cancelled, i.e. the path is an empty string, what happens? it seems like a success should be some kind of a tri-state instead of binary: ok, err, cancelled. although "cancelled" is very often indicated as a specific error in go.

Reply:
oh true! the backend in that case returns `nil/null` for the whole object in that case. will update the frontend accordingly

Reply:
done